packages/tools/devtools/devtools-example/src/App.tsx

```diff
@@ -93,6 +123,33 @@ async function populateRootMap(container: IFluidContainer): Promise<void> {
 	const sharedCounter = await container.create(SharedCounter);
 	rootMap.set(sharedCounterKey, sharedCounter.handle);
 
+	// Set up SharedTree for visualization
+	const sharedTreeFactoryObject = SharedTreeObject as unknown as SharedObjectClass<ISharedTree>;
+	const sharedTree = await container.create(sharedTreeFactoryObject);
+
+	// Populate SharedTree using Schema
+	const builder = new SchemaBuilder("SchemaAware");
+	const stringSchema = builder.primitive("string", ValueSchema.String);
+	const rootNodeSchema = builder.object("Test", {
+		local: {
+			children: SchemaBuilder.fieldSequence(stringSchema),
+		},
+	});
+	const schema = builder.intoDocumentSchema(
+		SchemaBuilder.field(FieldKinds.value, rootNodeSchema),
+	);
+
+	const view = sharedTree.schematize({
+		schema,
+		allowedSchemaModifications: AllowedUpdateType.None,
+		initialTree: { children: ["hello", "world"] },
+	});
+
+	// TODO: Console to avoid linter problem. Make sure to remove
+	console.log("Live Forever:", view.root);
```

Comment:
TODO :)

Reply:
That's embarrassing! Just in case you were curious, it was the song I was listening to when I was working on it 😅.

Reply:
I actually didn't even look at the console statement. I just saw the TODO and figured I would call it out. 😋